lib/regions.js

```diff
@@ -33,106 +30,180 @@ const update_region = query.bind(query, fs.readFileSync('./sql/update_region.sql
 const delete_region = query.bind(query, fs.readFileSync('./sql/delete_region.sql').toString('utf8'), to_response);
 
 async function list(pg_pool) {
-  return await select_regions(pg_pool, [])
+  return select_regions(pg_pool, []);
 }
 
-async function update(pg_pool, name, country, description, locale, private_capable, provider_name, provider_region, provider_availability_zones, high_availability, deprecated) {
+async function update(
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Async function 'update' has a complexity of 10. Maximum allowed is 8.](https://app.codacy.com/manual/Akkeris/controller-api/pullRequest?prid=5244943)